dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3GetJob.java

```diff
@@ -123,7 +123,9 @@ private void transferFromSelectedItem(final String bucketName,
         final FluentIterable<Ds3Object> ds3Objects = getDS3Objects(bucketName, selectedItem);
         final long totalJobSize = getTotalJobSize(ds3Objects);
         final Ds3ClientHelpers.Job job;
-
+        if (ds3Objects.isEmpty()) {
+            return;
```

Comment:
We should probably log something saying that we did not create the job due to there not being any jobs to transfer.